File: C#/Contoh Statement/Contoh If, Else If, dan Else/Nested If dan Else.cs
using System;

namespace Main
{
    class main
    {
        static void Main(string[] args)
        {
            string data1, data2;

            Console.WriteLine("Login :\n");
            Console.Write("username : ");
            data1 = Console.ReadLine();
            Console.Write("password : ");
            data2 = Console.ReadLine();

            if (data1 == "user1")
            {
                if (data2 == "psw123")
                {
                    Console.WriteLine("\nanda berhasil mencoba");
                }
                else
                {
                    Console.WriteLine("\npassword salah");
                }
            }
            else
            {
                Console.WriteLine("\nusername anda salah");
            }

        }
    }
}

File: Kotlin/Contoh Statement/Contoh If, Else If, dan Else/Nested If dan Else.kt
import java.util.Scanner

fun main(args: Array<String>) {
    var input = Scanner(System.`in`)

    println("Login \n")
    print("username : ")
    var data1 = input.nextLine()
    print("password : ")
    var data2 = input.nextLine()

    if (data1.equals("user1")){
        if (data2.equals("psw123")){
            println("\nanda berhasil mencoba");
        }
        else{
            println("\npassword salah");
        }
    }
    else {
        println("\niusername anda salah");
    }
}
